packages/v4/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -0,0 +1,168 @@
+
+To freshen things up, we made a number of changes, which can be seen in action over in our [empty state documentation.](/components/empty-state) These changes include the following: 
+
+- We replaced the `EmptyStatePrimary` and `EmptyStateSecondaryActions` components with `EmptyStateActions`.
+
+- We added the `EmptyStateHeader` component. To `EmptyStateHeader` you should pass an `EmptyStateIcon` to the `icon` property, a main title to `titleText` property, and (optionally) a title's heading level to the `headingLevel` property.
+
+- We added the `EmptyStateFooter` component that wraps any content following `EmptyStateBody`.
+
+### Helper text
+
+Our components often allow you to display descriptive or status text below a form field to help users make decisions, but the implementation of this text hasn’t been consistent across components. We wanted to clean things up so that this implementation is more consistent.
+
+As a result, a few components were updated to use the same [`HelperText` component](/components/helper-text), rather than their own disparate helper-text-like implementation. This change affects the following components:
+
+- [Date picker](/get-started/upgrade#date-picker)
+- [Time picker](/get-started/upgrade#time-picker)
+- [Form](/get-started/upgrade#form) 
+    - The `FormGroup` component is especially affected, as we've removed the following helper text related properties in PatternFly 5: 
+        - `helperText`
+        - `helperTextInvalid`
+        - `validated`
+        - `helperTextIcon`
+        - `helperTextInvalidIcon`
+        - `isHelperTextBeforeField`
+
+        To add helper text to a form group in place of these properties you should use the `FormHelperText`, `HelperText`, and `HelperTextItem` components directly as part of the `FormGroup`’s children. This implementation is shown in PatternFly's [invalid form example.](components/forms/form#invalid)
+
+### Input group
+
+Previously, the input group styles were too narrowly coupled with the types of children they contained. It did not match the approach we took to styling other groups of components. This update enabled more flexibility for the types of children allowed in an input group, and simplified the style rules for things like spacing, focus management, and item size manipulation.
+
+As a result, input group items no longer automatically fill the available space of their container. To retain previous styling, new `InputGroupItem` components may need to have the `isFill`, `isBox`, and/or `isPlain` properties adjusted. This is likely necessary for `FormSelect`, `TextInput`, `Select`, and `TextArea`, which may particularly require the `isFill` property to be passed to `InputGroupItem`.
+
+## Refactored components 
```

Comment:
I think using the word deprecated in a header so that it appears in the jump links might be important. Can we update this to `Refactored and deprecated components`?

Reply:
I think @mcarrano was wanting to avoid the use of the word "deprecated", but I may have misunderstood. I'll make this update for now though!

Reply:
I'm OK with saying "Refactored and deprecated." I mostly felt like it was important to say _why_ components were being deprecated to make sure we call attention to the value we are adding rather than what we are taking away.